person.rb

```diff
@@ -2,17 +2,18 @@
 require './decorator'
 
 class Person < Nameable
+  attr_accessor :name, :age
+  attr_reader :id, :rentals
```

Comment:
- [ ] Kindly move `:rentals` from `attr_reader` to `attr_accessor` as it is required to have `Getters` and `Setters` for rentals.

```suggestion
  attr_accessor :name, :age, :rentals
  attr_reader :id
```